api/app/reviews/api.py

```diff
@@ -238,4 +240,57 @@ def validate_scores(self, scores):
                 return self.get_error_message('value')
     
     def get_error_message(self, key):
-        return ({'message': {key: 'Missing required parameter in the JSON body or the post body or the query string'}}, 400)
\ No newline at end of file
+        return ({'message': {key: 'Missing required parameter in the JSON body or the post body or the query string'}}, 400)
+
+class ReviewAssignmentAPI(PostReviewAssignmentMixin, restful.Resource):
+    
+    @auth_required
+    def post(self):
+        args = self.post_req_parser.parse_args()
+        user_id = g.current_user['id']
+        event_id = args['event_id']
+        reviewer_user_email = args['reviewer_user_email']
+        num_reviews = args['num_reviews']
+
+        current_user_roles = self.get_roles(user_id, event_id)
+        if 'event_admin' not in current_user_roles and 'system_admin' not in current_user_roles:
+            return FORBIDDEN
+        
+        reviewer_user = self.get_reviewer_user(reviewer_user_email)
+        if reviewer_user is None:
+            return USER_NOT_FOUND

```

Comment:
Note this was an existing error message but it's returning 409 CONFLICT currently? Should I update this to be a 404? Not sure of the knock on effects on the front end though.

Reply:
@KaleabTessera I had a look at the `user.service.js` file and it looks like any error code is handled the same way. So changing it shouldn't affect the front end right?

Reply:
I think it should be fine to change to 404

Reply:
Done.